.azure-pipelines/pr_test_scripts.yaml

```diff
@@ -116,7 +116,7 @@ multi-asic-t1-lag:
   - iface_namingmode/test_iface_namingmode.py
   - scp/test_scp_copy.py
   - test_interfaces.py
-  - override_config_table/test_override_config_table.py
+#  - override_config_table/test_override_config_table.py
```

Comment:
thanks @yejianquan for this quick fix
can you please delete  `- tacacs/test_authorization.py` as well, it is quite flaky with ~20% fail rate even it can pass with rerun